frontend/coprs_frontend/tests/test_views/test_backend_ns/test_backend_general.py

```diff
@@ -11,6 +11,7 @@
 from coprs import app
 
 
+# pylint: disable=unused-argument
```

Comment:
We prefix the unused arguments with `_` typically.

Reply:
those arguments are fixtures so `_` prefix wont help here